### ISSUE TITLE
problem with install magenta

### ISSUE THREAD

**Issue by ghostPath** [tensorflow/magenta#864]
my python is python2.7  system is ubantu. When I follow the Manual Install, I got the error:
![9e519d6da0828a534f5a78a519bd918c](https://user-images.githubusercontent.com/5909556/30059292-0eacea9a-9271-11e7-90f6-3edc1d020550.jpg)

I have tried pip install ipython==5.4.0, but I got the same error!
Thanks for help!

**Comment by cghawthorne** [comment]
Can you provide some more details about the rest of your setup process? Were you using our automated install script or doing things manually? Did you use a conda environment?

**Comment by ghostPath** [comment]
@cghawthorne  I have tried that script, and I use conda environment.
![0502664ca9b1979e44955c5408b51257](https://user-images.githubusercontent.com/5909556/30091629-2a1a08c2-92ed-11e7-97ad-c8fb9022081c.jpg)
![b1c5f509201c5c19e34e3092d6d6f2fd](https://user-images.githubusercontent.com/5909556/30091638-364cdbf6-92ed-11e7-8002-bc36ee5ba00a.jpg)

**Comment by EvilMudkip** [comment]
Same problem here! I'm using Windows 10, downloaded Python 3.6 Miniconda, and entered the following code:
`conda create -n magenta python=3 jupyter` followed by
`activate magenta`, since source does not work.

I then proceeded to enter: `pip install magenta`, and got the exact same error.

**Comment by ghostPath** [comment]
@EvilMudkip Have you sloved this problem?

**Comment by EvilMudkip** [comment]
@ghostPath Yeah, after I uninstalled everything and reinstalled them numerous times haha.

Thanks!
